packages/flutter/test/widgets/editable_text_test.dart

```diff
@@ -6619,6 +6629,7 @@ void main() {
     final EditableTextState state = tester.state<EditableTextState>(find.byType(EditableText));
     state.updateEditingValue(const TextEditingValue(
       text: 'foo composing bar',
+      selection: TextSelection.collapsed(offset: 4),
```

Comment:
selection needed to be within composing range, otherwise the composing will be reseted by texteditingcontroller. It is not needed before because the it does not fire selection changes prior to this pr